### DISCUSSION TITLE
npx git-cz does not detect changed packages

### DISCUSSION THREAD

**Discussion by phun-ky** [lerna/lerna#3729] [Q&A]
After adding some changes, I run `npx git-cz` to do the commits, but the list of packages displayed says that the change did not affect any packages. How do I fix this? I have this in my root `package.json`:

```json
{
  "workspaces": [
    "packages/utils/*",
    "packages/routes/*",
    "packages/middlewares/*"
  ]
}
```

**Comment by fahslaj** [comment]
Hi @phun-ky, I am sorry that you are experiencing this issue. Can you please elaborate on how this is related to Lerna?

**Reply by phun-ky** [reply]
Well, I have used `lerna` before it changed ownership, and I had _no_ problem with using `lerna` and `git-cz` together, where it detected the changes in the packages. Since `git-cz` has to read the packages to look for changes, depending on the `lerna` setup, and it worked before, it seems natural to start here asking.

**Comment by phun-ky** [comment]
Nevermind, through trial and error with different commitizen adapters, i had to switch to to `git-cz` using `git cz` as a script to get the scopes correctly. ignore this :duck:
